### PR TITLE
fix(oracle): change rate of price modulations 

### PR DESCRIPTION
**Pull request by n00b21337**
Was done on top of the branch where we have skipped rounds included.
Reworked tests to reflect the changes.

### CODE REVIEW

File: test/util/tools.ts [hunk 1]
```diff
@@ -22,14 +22,18 @@ async function getBlockNumber(): Promise<number> {
   return parseInt(blockNumber);
 }
 
-async function skippedRoundsIncrease(n: number, price: number): Promise<number> {
-  const maxIncreaseRate = 1036;
-  const multiplier = 1024;
+async function skippedRoundsIncrease(
+  n: number,
+  price: number,
+  priceBaseString: number,
```
Comment (nugaon) [inline test/util/tools.ts:28]: why is it named `priceBaseString` when it is a number?

File: test/util/tools.ts [hunk 2]
```diff
@@ -22,14 +22,18 @@ async function getBlockNumber(): Promise<number> {
...
+  maxIncreaseRate: number
+): Promise<number> {
+  const base = priceBaseString;
```
Comment (nugaon) [inline test/util/tools.ts:31]: just called it `base` in the argument definitions, this assignment does not do anything than just renaming the variable

File: test/Redistribution.test.ts [hunk 3]
```diff
@@ -688,6 +689,10 @@ describe('Redistribution', function () {
           r_node_1 = await ethers.getContract('Redistribution', node_1);
           r_node_2 = await ethers.getContract('Redistribution', node_2);
 
+          // Set price base
+          const priceBase = await priceOracle.priceBase();
+          priceBaseString = priceBase.toString();
```
Comment (nugaon) [inline test/Redistribution.test.ts:694]: it is string by default, isn't it?

File: test/Redistribution.test.ts [hunk 4]
```diff
@@ -873,6 +883,10 @@ describe('Redistribution', function () {
           priceOracle = await ethers.getContract('PriceOracle', deployer);
           await priceOracle.unPause(); // TODO: remove when price oracle is not paused by default.
 
+          // Set price base
+          const priceBase = await priceOracle.priceBase();
+          priceBaseString = priceBase.toString();
```
Comment (nugaon) [inline test/Redistribution.test.ts:888]: you cast it to int at each occurrence, why don't you just cast it here to int instead?

**Review by nugaon — APPROVED** [review]
Overall the main changes seem correct

File: test/Redistribution.test.ts (hunk 4 above)
Comment (n00b21337) [reply test/Redistribution.test.ts:888]: was modeled after
 let minPriceString: string;
 
 as it derived from there as a new separate value. That needed a string for setPrice method, but this one doesn't and you are right, it shouldn't be cast back and forth. I will improve that part. thnx

File: test/Redistribution.test.ts (hunk 3 above)
Comment (n00b21337) [reply test/Redistribution.test.ts:695]: It is not, it's an Object. BigNumber object, so it needs to be cast down to something but this was not needed to be string as we just use the number, so good call on this

**Comment by n00b21337** [comment]
> Overall the main changes seem correct

thnx for the review, good call on tests, the code was sub-optimal and also had some bad namings. Improved all of that and more, tests run smoothly but are more clear. Will merge tomorrow if you do not have anything other to add.

**Review by mrekucci — APPROVED** [review]
Except for the reported lint issues about unused variables. LGTM!

**Pull request merged**